### DISCUSSION TITLE
No domain verification challenge

### DISCUSSION THREAD

**Discussion by raiadi2008** [vercel/community#3805] [Help]
### Summary

const vercelDomainVerifyEndpoint = `${process.env.VERCEL_URL}/v10/projects/${process.env.VERCEL_PROJECT_ID}/domains/${domain}/verify?teamId=${process.env.VERCEL_TEAM_ID}`

i am using this endpoint to fetch if the domain is verified. Now the domain is not verified be it always returns verified response

`{
    "name": "www.mydomain.co",
    "apexName": "mydomain.co",
    "projectId": "my_project_id",
    "redirect": null,
    "redirectStatusCode": null,
    "gitBranch": "dev",
    "updatedAt": 1693222689806,
    "createdAt": 1693222689806,
    "verified": true
}`

### Steps to Reproduce

OS: Mac OS
ENV: Local
PRO Team subsccription

**Comment by amyegan** [comment]
Hey @raiadi20086. When you say the domain is not verified, what message do you see in project settings for that domain?  Does it ask you to add a TXT record, or does it say that it needs an A record to be added?

**Reply by raiadi2008** [reply]
It says invalid configuration. And say to add A record and CNAME record

**Reply by amyegan** [reply]
Whether `verified` only returns `false` in situations where a TXT record is needed to [verify domain access](https://vercel.com/docs/projects/domains/add-a-domain#verify-domain-access). This most often happens if a domain is already associated with a different team or account. 

You always need to use an A record or change nameservers to [configure a custom domain](https://vercel.com/docs/projects/domains/add-a-domain#). You can use [`/v6/domains/{domain}/config`](https://vercel.com/docs/rest-api/endpoints#get-a-domain-s-configuration) to check domain configuration.
